Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -382,21 +401,22 @@ def update_with_tests_sections(missing_ids, modified_files, test_ids, tests):
     for file_path in modified_files:
         tests_from_file = get_tests(file_path)
         for test in tests_from_file:
-            if test in test_ids or re.match(collect_helpers.NO_TESTS_FORMAT, test, re.IGNORECASE):
-                if collect_helpers.checked_type(file_path, collect_helpers.INTEGRATION_REGEXES):
-                    _id = tools.get_script_or_integration_id(file_path)
+            if collect_helpers.checked_type(file_path, collect_helpers.INTEGRATION_REGEXES):
+                _id = tools.get_script_or_integration_id(file_path)
 
-                else:
-                    _id = get_name(file_path)
+            else:
+                _id = get_name(file_path)
+            if test in test_ids or re.match(collect_helpers.NO_TESTS_FORMAT, test, re.IGNORECASE):
```

Comment:
Maybe it is clear but can you elaborate on why we have moved this line to here?

Reply:
By this line i guess you mean getting the _id.
This is because in my else clause in lines 418-419 i need to remove him from missing ID set if pack is not xsoar supported
                else:
                    missing_ids = missing_ids - {_id}

Reply:
@ShahafBenYakir Let me know if you are okay with that and i can resolve conversation

Reply:
No I mean the `if test..` line (when I mark one line in Github it always displays the 3-4 lines above it as well)

Reply:
?